src/components/connection_handler/src/heartbeat_monitor.cc

```diff
@@ -90,18 +90,18 @@ void HeartBeatMonitor::threadMain() {
 }
 
 void HeartBeatMonitor::AddSession(uint8_t session_id) {
-  LOG4CXX_DEBUG(logger_,
-                "Add session with id " << static_cast<int32_t>(session_id));
+  const uint32_t converted_session_id = static_cast<int32_t>(session_id);
```

Comment:
@AByzhynar This variable used only in logger, Please mark it as UNUSED

Reply:
@LuxoftAKutsan Done in 8033d26b57aa342fcd191c58bcac5a46c00a649a